config.js

```diff
@@ -825,10 +825,10 @@ config.ALLOW_HTTP = false;
 config.BASE_MODE_CONFIG_FILE = 0o600;
 config.BASE_MODE_CONFIG_DIR = 0o700;
 
-config.NSFS_WHITELIST = [];
+config.NC_WHITELIST = [];
```

Comment:
this actually should be S3_WHITELIST as we check it for any S3 request, not just NSFS or NC.

Reply:
changed to S3_CLIENT_IP_WHITELIST as we agreed on